pa11ycrawler/pipelines.py

```diff
@@ -182,6 +221,7 @@ def process_item(self, item, spider):
                 )
             )
 
+        self.check_title_match(item['page_title'], stdout, spider.logger)
```

Comment:
basic question: when check_title_match encounters an error, will something be raised (e.g., skip the subsequent lines of code)?